src/azure-cli/azure/cli/command_modules/vm/generated/_help.py

```diff
@@ -52,3 +57,75 @@
     type: command
     short-summary: "Delete an SSH public key."
 """
+
+helps['sig shared-gallery'] = """
+    type: group
+    short-summary: Manage shared gallery with vm
+"""
+
+helps['sig shared-gallery list'] = """
+    type: command
+    short-summary: "List shared galleries by subscription id or tenant id."
+    examples:
+      - name: Get a gallery.
```

Comment:
`List shared galleries`?

Reply:
I override the help info in manual help file.
We should suggest the service team to improve their Swagger quality because this name came from Swagger definition